ArduCopter/Parameters.cpp

```diff
@@ -939,14 +939,22 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Path: mode_systemid.cpp
     AP_SUBGROUPPTR(mode_systemid_ptr, "SID", 34, ParametersG2, ModeSystemId),
 #endif
-    
+
     // @Param: FS_VIBE_ENABLE
     // @DisplayName: Vibration Failsafe enable
     // @Description: This enables the vibration failsafe which will use modified altitude estimation and control during high vibrations
     // @Values: 0:Disabled, 1:Enabled
     // @User: Standard
     AP_GROUPINFO("FS_VIBE_ENABLE", 35, ParametersG2, fs_vibe_enabled, 1),
 
+    // @Param: FS_OPTIONS
+    // @DisplayName: Failsafe options bitmask
+    // @Description: Bitmask of additional options for battery, radio, & GCS failsafes. 0 (default) disables all options.
+    // @Values: 0:Disabled, 1:Continue in auto mode on RC FS only, 2:Continue auto mode on GCS FS only, 3:Continue in auto mode on RC and/or GCS FS, 19:Continue in auto mode on RC and/or GCS FS + continue in pilot controlled modes
+    // @Bitmask: 0:Continue if in auto mode on RC failsafe, 1:Continue if in auto mode on GCS failsafe, 3:Continue if in guided mode on RC failsafe, 4:Continue if landing on any failsafe, 5:Continue if in pilot controlled modes on RC failsafe
```

Comment:
For the FS_OPTIONS parameter Bitmask, is "5" actually "Continue if in pilot controlled modes on GCS failsafe" (instead of RC?").  I've added a fix along with some other slight wording changes to my branch